setup.py

```diff
@@ -58,7 +59,6 @@ def get_package_data(package):
     package_data=get_package_data("defender"),
     install_requires=["Django>=1.8,<3.2", "redis<3.6"],
```

Comment:
Add `python_requires`? I've had comments from users before about releasing versions which drop old Python versions without this. If they don't have versions pinned they get upgraded to a (known) failing version.

Reply:
Good point :+1: . I added `python_requires`